mobile/native/src/db/mod.rs

```diff
@@ -347,7 +348,35 @@ pub fn get_spendable_outputs(
         .map(|output| output.try_into())
         .collect::<Result<Vec<_>>>()?;
 
-    tracing::info!(?outputs, "Got all spendable outputs");
+    tracing::debug!(?outputs, "Got all spendable outputs");
 
     Ok(outputs)
 }
+
+pub fn upsert_channel(channel: ln_dlc_node::Channel) -> Result<()> {
+    tracing::debug!(?channel, "Inserting channel");
```

Comment:
Was this really necessary or could you just have faked this for the App? If it is not being used/needed in the app, you could have simply had an empty funciton here.

Reply:
well, I guess faking it would have worked, but I think having a shadow of the channel beyond closing a channel is also useful to the app. However, I am not invested into having this already implemented with this change. I am happy to remove this for the app side. I leave it up to the reviewers to decide.

Reply:
If we keep it I see us having to maintain it in the future because the db model changed, etc.

> I leave it up to the reviewers to decide.

I'll leave it up to you :P